config/initializers/mongo.rb

```diff
@@ -0,0 +1,17 @@
+module QME
+  patient_cache = Mongoid.default_client['patient_cache']
+  # create a unique index for patient cache, this prevents a race condition where the same patient can be entered multiple times for a patient
+  patient_cache.indexes.create_one({ 'value.measure_id' => 1, 'value.sub_id' => 1, 'value.effective_date' => 1, 'value.patient_id' => 1 },
```

Comment:
it was `create` in the original from cypress v2, but it threw an error `undefined method 'create' for #<Mongo::Index ...`
I changed it to `create_one` after seeing it here: https://docs.mongodb.org/ecosystem/tutorial/ruby-driver-tutorial/#indexing